### DISCUSSION TITLE
mac上有更好用的客户端吗

### DISCUSSION THREAD

**Discussion by Jarvis110469** [XTLS/Xray-core#700] [Q&A]
qv2ray的分流非常难用，明明设置本地地址绕行，他还是会强行代理，电报等各类软件也都存在无法连接服务器的情况，Windows上的v2rayn就没有这种问题，求各位大神推荐其他更好用的客户端

**Comment by ghost** [comment]
@fanxinghaoyue0 shadowrocket(x)

**Comment by Jarvis110469** [comment]
> @fanxinghaoyue0 shadowrocket(x)

mac上没有这个啊

**Comment by ghost** [comment]
@fanxinghaoyue0 支持啊，去年就可以了

**Comment by Jarvis110469** [comment]
> @fanxinghaoyue0 支持啊，去年就可以了

嗯 但是mac不支持这个软件啊

**Comment by ghost** [comment]
@fanxinghaoyue0 哦，你不是m1的mac啊，好像确实没有什么好的办法ㄌ

**Comment by dracher** [comment]
可以试下 [clashx](https://github.com/yichengchen/clashX/releases)

**Comment by yrong** [comment]
qv2ray

**Comment by SekiBetu** [comment]
~~装Windows~~

**Comment by fsiisx** [comment]
有钱surge，没钱clashx pro
